tensorboard/webapp/runs/views/runs_table/runs_data_table.scss

```diff
@@ -46,6 +46,10 @@ $_arrow_size: 16px;
   width: 100%;
 }
 
+.selected-checkbox {
```

Comment:
Personally I would prefer if the "header" element was given z-index: 1 to declare that it should always be shown atop the content of other rows. This would also reduce likelyhood of a similar bug being introduced later (for example if we added another checkbox column in the runs or another table).